src/host/host.cpp

```diff
@@ -11,8 +11,8 @@
 #include <thrift/transport/TBufferTransports.h>
 #include <thrift/transport/TServerSocket.h>
 
-#include "../constants/constants.h"
-#include "../constants/shared.h"
+#include "constants.h"
+#include "shared.h"
 #include "../errors/errors.h"
```

Comment:
Just a thought, do we want to move `errors.h` into libcommon? Anyway we could have the include path be `#include errors.h"`?

Reply:
This makes sense to me, I was thinking of moving it into the host since client and host are generally independent, but I'm happy to move it into `libcommon` if desired